wiki/Glossary/en.md

```diff
@@ -352,34 +350,31 @@ For more information about the Extended Chat Console, see [Chat Console](/wiki/C
 Fail has three meanings:
 
 - the incompletion of a beatmap due to:
-  - not filling the healthbar up to 80% when completing a beatmap (osu!taiko)
-  - empty healthbar (osu!standard, osu!catch, and osu!mania)
+  - not filling the healthbar up to 80% when completing a beatmap (in [osu!taiko](/wiki/osu!taiko))
+  - empty healthbar ([osu!standard](/wiki/osu!standard), [osu!catch](/wiki/osu!catch), and [osu!mania](/wiki/osu!mania))
 - in multiplayer, an indicator underneath the player's name to show they have failed the beatmap
 - the player having less than 50% health during a break; where an indicator will display if you failed the section
   - a fail indicator will be shown
   - a failed storyboard animation will be played instead (if available).
 
 ### FC
 
-![](img/CombosCanHurt.png)
+![](img/CombosCanHurt.png "The score on the top has a full combo, while the one on the bottom has a broken combo.")
```

Comment:
`![Comparison between full combo and broken combo.](img/CombosCanHurt.png "The score on the top has a full combo, while the one on the bottom has a broken combo.")`